torch_xla/csrc/ops/triangular_solve.cpp

```diff
@@ -0,0 +1,106 @@
+#include "torch_xla/csrc/ops/triangular_solve.h"
+
+#include "tensorflow/compiler/xla/client/xla_builder.h"
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+// This function plays two roles:
+// - computes the output shape
```

Comment:
Computes ...

Sentences begin with Capital, and ends with dot (.).

Reply:
Sorry about that. I will keep this in mind henceforth.